code/game/objects/items/handcuffs.dm

```diff
@@ -377,7 +378,7 @@
 	icon_state = "bola_r"
 	item_state = "bola_r"
 	breakouttime = 70
-	knockdown = 20
+	knockdown = 35
```

Comment:
Undocumented change

Reply:
I can explain, so basically I was working on this in a convoluted and dumb way but found the right way and copy pasted code but it appears I might have forgot to change back some of the stuff. Will fix it, sorry for that.

Reply:
should be fine now